api/wd/index.md

```diff
@@ -37,48 +37,43 @@ before(function requiresAlerts() {
 });
 ```
 
-#### `browser.getConsoleLogs(logLevel='all')`
+### `browser.getConsoleLogs([logLevel: string]): Promise<Object>`
 
-Returns all log events with `logLevel` (log/warn/error/debug) since the last time this method was called.
+Returns all log events for `logLevel` `all`(default) `log`,`warn`, `error`, `debug` since the last time this method was called.
 **Warning:** Each browser implements this differently against the WebDriver spec.
 
 ```js
-var errorLogs = browser.getConsoleLogs('error');
+const errorLogs = browser.getConsoleLogs('error').then(errorLogs => { /* ... */});
 ```
 
-#### `browser.getScreenshot()`
+### `browser.getScreenshot(): Promise<string>`
 
 Returns screenshot as a base64 encoded PNG.
 
-#### `browser.assertImgLoaded(selector)`
+## Navigation
 
-*Not implemented yet.*
-
-Asserts that the image element at `selector` has both loaded and been decoded successfully.
-
-```js
-browser.assertImgLoaded('.logo');
-```
-
-### Navigation
-
-#### `browser.getHeader(name)`
+### `browser.getHeader(name: string): Promise<any>`
```

Comment:
i assume this is really something like `: Promise<string | string[]>` - can we check if this is correct and put the real type?